src/contributor-guide/contributing_dod.md

```diff
@@ -14,32 +14,31 @@ The requirements that result from our DoD are applied by default to all Magento-
 Internally, the Magento development teams follow the DoD to ensure that all work meets consistent release criteria.
 We encourage our community contributors to also follow the Magento DoD.
 
-At a high level, the criteria defined in our "definition of done" revolve around the principle of writing clean code that is reviewed and includes unit tests and solid documentation along with the code.
+At a high level, the criteria defined in our "definition of done" revolve around the principle of writing clean code that is reviewed and includes automated tests and solid documentation along with the code.
 
-The following sections provide additional details about each of these criteria:
+## Definition of Done
 
-## Backward Compatibility
+The checklist that can be used to evaluate if the DoD has been met:
 
-### Functional Backward Compatibility
+-  All changes must be approved by reviewer.
+-  All changes must be covered by automated tests if that is possible.
+-  All functional changes should be documented.
+-  All backward incompatible changes should be approved and covered by static tests if that is possible. Static tests should be delivered to the [magento-coding-standard][5] repository.
+-  TCO impact should be defined for all changes.
 
-**Functional backward compatibility** means the behaviour of the application is preserved.
+The following sections provide additional details about each of these criteria:
 
-Existing product features and functionality must be retained during any changes to the code.
-Any backward-incompatible functional changes must be approved by a product owner.
-The documentation should explain the justification and provide the "business value".
+## Review
 
-### Technical Backward Compatibility
+A task, when complete by the author, should be reviewed by a maintainer through a process of formal code review.
 
-**Technical backward compatibility** means the technical interfaces are preserved.
+The reviewer should check whether the task meets the original acceptance criteria, verify that there are no code defects and that all points of this definition of done are met.
 
-Technical interfaces include PHP interfaces or classes, CLI commands, URLs or any other interfaces that can be used by 3rd-party developer, system integrator, or user of Magento.
-Any change to an interface that can lead to a broken integration is a breaking technical change.
-Technical interfaces and the corresponding level of change is described in [Code Changes][6] and [Module Version Dependencies][7] documents in more details.
+## Tests
 
-Technical backward compatibility must be retained between PATCH (marketing) versions of Magento products. It should also be retained between MINOR (marketing) releases if possible.
-Any breaking changes must be approved by an architect, product owner and release manager.
+All changes to Adobe Commerce projects should be covered by automated tests to meet the "Definition of Done" (DoD).
```

Comment:
should  a line be added to say that manual tests should be provided when automated tests cannot cover the change?

should we also mention that the manual testing done for verifying the change(s) should cover the scope of all potentially impacted areas?

Reply:
I have extracted the automated tests section to a separate page preserving the content. DoD did not contain mentions of manual testing. I think mentioning it is a good idea, but it should be proposed in a separate PR.